_docs/_latest/develop/dev/scanner.md

```diff
@@ -51,8 +52,8 @@ To design a rules engine, follow the guidelines below:
     *   The rule book is a data structure that provides a way to compare a rule
         with a policy.
     *   The `find_policy_violations()` method searches the rule book and
-        compares the policy against the rule (if found) in the book.
+        compares the policy against the rule in the book, if found.
 1. Create a new scanner and add the necessary scanner mappings and configurations.
 
-Refer to `scanner.py` for an example of how the `IamRulesEngine` works with the 
-other Forseti scanners.
+For an example of how the `IamRulesEngine` works with the other Forseti scanners,
+see `scanner.py`.
```

Comment:
@joecheuk and @blueandgold can we do any better than just saying "look for and find scanner.py"? What am I looking for, maybe we should link to the generated pydoc instead?

Reply:
Perhaps we can say this:
```
...see how scanner.py determines and initializes which scanners to run.
```

Reply:
That doesn't really answer the question of "see how", I think we need to be much more explanatory.

Reply:
On further reflection, this sentence is out of place.  The scanner.py has nothing to do with any specific rules engine.  All the scanner.py does is to find which scanners are configured to run, and then run them.  So, this sentence about IamRulesEngine in scanner.py can be deleted.  I have doubled-checked with joe and he didn't write this and feels this can be removed as well.

Reply:
Resolved in d06787fa